contracts/Contribution.sol

```diff
@@ -16,6 +16,10 @@ contract Contribution is Controlled, TokenController {
   uint256 public totalSupplyCap;            // Total AIX supply to be generated
   uint256 public totalSold;                 // How much tokens have been sold
 
+  uint256 public numWhitelistedInvestors;
+  mapping (address => bool) public canPurchase;
+  mapping (address => uint256) public individualSold;
```

Comment:
I thought there could be something like:
```
  struct WhitelistedInvestor {
      uint256 canPurchase;
      uint256 individualSold;
  }
  mapping(address => WhitelistedInvestor) investors;
```

Reply:
I feel like these are 2 completely different and kind of unrelated values.
I don't see any benefit of creating a structure every time.

Still is a start